packs/data/feats.db/tengu-feather-fan.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>You've learned to bind some of your feathers together into a fan to focus your ancestral magic. You gain a <em>tengu feather fan</em>: a magic item of light Bulk with a level equal to your level and the evocation and primal traits. You (and only you) can use the <em>tengu feather fan</em> to cast tengu magic. If your fan is lost or destroyed, you can create a replacement during your daily preparations; if you do so, your previous fan falls apart into mundane feathers. Further feats might grant you additional activations of your <em>tengu feather fan</em>, but you can never have more than three activations per day, no matter how many such feats you have. The DC for spells you cast with your <em>tengu feather fan</em> is your class DC or spell DC, whichever is higher.You can cast the 1st-level <em>@Compendium[pf2e.spells-srd.Gust of Wind]{Gust of Wind}</em> spell by activating your <em>tengu feather fan</em>.</p>\n<hr />\n<p><strong>Activate</strong> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You cast one of the spells contained in your t<em>engu feather fan</em>. Activating the fan takes the spell's normal number of actions. You can also Activate the fan to cast a cantrip you've gained from a heritage or ancestry feat; this activation doesn't count against the fan's uses per day.</p>"
+            "value": "<p>You've learned to bind some of your feathers together into a fan to focus your ancestral magic. You gain a <em>tengu feather fan</em>: a magic item of light Bulk with a level equal to your level and the evocation and primal traits. You (and only you) can use the <em>tengu feather fan</em> to cast tengu magic. If your fan is lost or destroyed, you can create a replacement during your daily preparations; if you do so, your previous fan falls apart into mundane feathers. Further feats might grant you additional activations of your <em>tengu feather fan</em>, but you can never have more than three activations per day, no matter how many such feats you have. The DC for spells you cast with your <em>tengu feather fan</em> is your class DC or spell DC, whichever is higher. You can cast the 1st-level <em>@Compendium[pf2e.spells-srd.Gust of Wind]{Gust of Wind}</em> spell by activating your <em>tengu feather fan</em>. @Check[type:fortitude|dc:resolve(@actor.attributes.classOrSpellDC.value)|traits:air]{Fortitude saving throw}</p>\n<hr />\n<p><strong>Activate</strong> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You cast one of the spells contained in your t<em>engu feather fan</em>. Activating the fan takes the spell's normal number of actions. You can also Activate the fan to cast a cantrip you've gained from a heritage or ancestry feat; this activation doesn't count against the fan's uses per day.</p>"
```

Comment:
Need to add the other traits to the inline, `evocation,item:trait:spell,primal`

Will need to do this for all of them.